backend/src/main/java/com/team4/backend/mapping/MonitorMapper.java

```diff
@@ -1,20 +1,33 @@
 package com.team4.backend.mapping;
 
+import com.team4.backend.dto.MonitorDto;
 import com.team4.backend.model.Monitor;
 
 public abstract class MonitorMapper {
 
-    //TODO add tests
+    public static Monitor toEntity(MonitorDto monitorDto) {
+        return Monitor.monitorBuilder()
+                .id(monitorDto.getId())
```

Comment:
id coming from frontend shouldn't be trusted -> remove id mapping inside toEntity